components/LeaderboardPage/LeaderboardPage.tsx

```diff
@@ -1,37 +1,39 @@
 import React, { useState, useEffect } from "react"
-import * as _ from "lodash"
-import { useMediaQuery } from "usehooks-ts"
-import getOwnersForCollection from "../../lib/alchemy/getOwnersForCollection"
-import getParticipants from "../../lib/getParticipants"
 import LeaderboardRow from "./LeaderboardRow"
 import SkeletonTableBody from "./SkeletonTableBody"
 import Layout from "../Layout"
+import getReferralLeaderboard from "../../lib/getReferralLeaderboard"
+import { getOwnersofCre8ors } from "../../lib/cre8or"
+import getTwitterHandleByAddress from "../../lib/getTwitterHandleByAddress"
 
 const LeaderboardPage = () => {
   const [collectors, setCollectors] = useState([])
-  const isMobile = useMediaQuery("(max-width: 768px)")
 
   useEffect(() => {
     const fetchTopCollectors = async () => {
```

Comment:
we're not getting collectors anymore right?

might be good to refactor this fx name ✏️ 

```suggestion
    const fetchLeaderboard = async () => {
```

Reply:
updated ✅